ghost/core/test/e2e-api/members/send-magic-link.test.js

```
@@ -5,7 +5,7 @@ const assert = require('assert/strict');
 const settingsCache = require('../../../core/shared/settings-cache');
 const settingsService = require('../../../core/server/services/settings');
 const DomainEvents = require('@tryghost/domain-events');
-const {anyErrorId} = matchers;
+const {anyErrorId, anyString, anyObject} = matchers;
```

ESLint

[error] 'anyObject' is assigned a value but never used. Allowed unused vars must match /^should$/u.

(no-unused-vars)